DLaB.Xrm.Core/Plugin/IExtendedPluginContext.cs

```diff
@@ -29,6 +29,11 @@ public interface IExtendedPluginContext : IPluginExecutionContext, IExtendedExec
         /// </value>
         IServiceProvider ServiceProvider { get; }
 
+        /// <summary>
+        /// Managed Identity Service for things not accounted for in the base class.
+        /// </summary>
+        new IManagedIdentityService ManagedIdentityService { get; }
```

Comment:
Why is there a "new" prefix?

Reply:
fixed, I was getting errors not using the new until I changed to using the _managedIdentityService = serviceProvider.Get<IManagedIdentityService>();